autoinstall/Pardus/preseed.cfg

```diff
@@ -0,0 +1,250 @@
+#_preseed_V1
+#### Contents of the preconfiguration file (for pardus)
+
+d-i debconf/language string en_US:en
+d-i debian-installer/country string US
+d-i debian-installer/fallbacklocale string en_US
+d-i debian-installer/language string en_US:en
+d-i languagechooser/locale string en
```

Comment:
These configs are also related to language, country and locale, why can't you put them under Localization?

Reply:
From testing result, the autoinstall for pardus will stuck at "Select language" if without these items